### PR TITLE
Adjustable Line Spacing

### LINKED ISSUE #25: [Feature] Configurable Leading
### Feature Description

Allow users to adjust the line spacing (Leading) of the displayed fonts.

### Reason for Change

This allows the user to adjust the line spacing of the fonts they are looking at to be able to see the text closer to how it would be used in production.

### Where are the changes needed?

The changes will be made to the components that hold the fonts, as well as new files to be able to adjust it.

### Code Snippets

_No response_

### Anything Else?

_No response_

### PR DESCRIPTION
**Pull request by cste303**
### Changes

Describe the Pull Request here. Add any references and info to help reviewers understand your changes.

- Added ability for users to adjust line spacing

### Tests applied

What testing has been applied to validate changes are accurate and correct

- Manual testing

### Issue ticket number(s)

Enter the issue numbers resolved by this pull request

- Resolves #25 

### Checklist

- [x] Documented
- [x] Linting tests successful
- [x] merge updated prod branch into development branch

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -32,9 +38,23 @@ function App() {
...
+          <div>
           <FontUploader onFontSelected={handleFontSelectedLeft} />
-          {selectedFontLeft && <FontPreview fontFile={selectedFontLeft} side="left" />}
+          <div className="line-height-adjustment">
+            <label htmlFor="lineHeightInputLeft">Adjust Line Height:</label>
+            <input
+              type="number"
+              id="lineHeightInputLeft"
+              value={lineHeightLeft}
+              step={0.1}
```
Comment (jaskfla) [inline src/App.tsx]: Steps of 0.1 are pretty massive when it comes to line height adjustments. If we consider that our “default” line height is 1.5, that’s more than a 5% increment in one go.

Perhaps 0.05 or even 0.025?

Ditto for the right-hand step.

**Review by jaskfla — COMMENTED** [review]
I’m still not confident making judgements about TypeScript, so am mostly just commenting on stylistic/design decisions here.

I’ve made a small (but important) suggestion regarding line height adjustments.

File: src/App.tsx (hunk 1 above)
Comment (jaskfla) [reply src/App.tsx]: I suppose a slider going between roughly 0.85 to maybe 2.0 would be ideal, but I’m not sure how much trouble that would be to implement.

**Comment by hwan513** [comment]
Seems like formatting for the `App.tsx` file is not being applied. Make sure Prettier is set up in your code editor and formatting on save works for that, since your git hooks might not be working correctly. Otherwise, run `pnpm run format` before creating the PR.

File: src/App.tsx (hunk 1 above)
Comment (cste303) [reply src/App.tsx]: Not 100% sure how to do a slider. I will make the step count 0.05 for now though.

Comment (jaskfla) [reply src/App.tsx]: This’ll do—thanks!

**Review by jaskfla — APPROVED** [review]
Looks good; I’ll go a head and merge.

**Pull request merged**